### DISCUSSION TITLE
Welcome to obsidian-prozen Discussions!

### DISCUSSION THREAD

**Discussion by cmoskvitin** [cmoskvitin/obsidian-prozen#5] [Announcements]
Hi! Here you can leave your feedback and ideas on how to improve the plugin.

**Comment by blackpleather** [comment]
I'm a new user of Obsidian (macOS) and I just downloaded and installed it from Community Plugins. I do not see how to invoke ProZen mode, nor is it mentioned in the ReadMe doc. What am I missing? Thanks.
